std/algorithm/searching.d

```diff
@@ -207,6 +202,38 @@ evaluate to true.
     assert(any!"a == 2"(a));
 }
 
+/++
+Checks if $(I _none) of the elements verify `pred`. This is a convenience wrapper for `!any!pred`.
```

Comment:
Auto-highlighting has been removed recently - `_`  is no longer necessary :)
https://github.com/dlang/dlang.org/pull/2307